models/news.models.js

```diff
@@ -0,0 +1,11 @@
+const db = require("../db/connection");
+
+exports.fetchTopics = () => {
+    return db.query("SELECT * FROM topics;").then((topics) => {
+        const topicsData = topics.rows;
+        if (!topicsData) {
+            return Promise.reject({ status: 404, msg: "Not Found - Invalid Path"});
```

Comment:
I don't think this error is needed. If the path was invalid, we would have never gotten into the model in the first place. 

If there is no topicData, I think that sending back an empty array would suffice as the path is fine